### PR TITLE
Show all Scaladoc options

### PR DESCRIPTION
**Pull request by som-snytt**
Demonstrate the styles of indentation without prejudice.

### CODE REVIEW

File: style/scaladoc.md [hunk 1]
```diff
@@ -10,43 +10,65 @@ previous-page: declarations
...
      *
-     *  This is further documentation of what we're documenting.  It should
-     *  provide more details as to how this works and what it does.
+     * This is further documentation of what we're documenting.
+     * Here are more details about how it works and what it does.
      */
-    def myMethod = {}
+    def member: Unit = ()
+
+Scaladoc style, with gutter asterisks aligned in column two:
```
Comment (olafurpg) [inline style/scaladoc.md]: s/ScaLadoc/Scaladoc/ ?

**Comment by eddsteel** [comment]
To fix the issue where tools/editors refer to "Scaladoc style", could these have names? I've seen flat and ragged used elsewhere. Thus your options could be "Javadoc", "Flat Scaladoc", "Ragged Scaladoc".

**Comment by olafurpg** [comment]
I totally agree @eddsteel, if the official style will advertise three different styles then it should give each style a name. However, I think we should first try to see if we can agree on a single style first and avoid the need for coming up with three names. Let's continue this discussion in https://contributors.scala-lang.org/t/quiet-changes-to-the-style-guide/923/6

**Comment by SethTisue** [comment]
hey @som-snytt, rebase? and then let's get this merged. in the absence of any broader changes to how the style guide is managed, in the meantime we might as well merge this, it reflects the current status quo

**Comment by SethTisue** [comment]
thanks, A.P.!

**Comment by som-snytt** [comment]
@SethTisue N.P.

**Pull request merged**